src/remembrall/classes/Note_class.py

```diff
@@ -0,0 +1,64 @@
+import datetime
+from collections import UserDict
+from remembrall.classes.customErrors import NoteError
+
+
+class Note:
+    def __init__(self, title: str, content: str):
+        self.title = title
+        self.content = content
+        self.tags = list()
+        self.creation_date = datetime.datetime.now()
+    
+    def add_tag(self, tag):
+        if tag not in self.tags:
+            self.tags.append(tag)
+    
+    def remove_tag(self, tag):
+        if tag in self.tags:
+            self.tags.remove(tag)
+        else:
+            raise NameError
```

Comment:
гадаю тут інший вийняток
```suggestion
            raise NoteError
```